app/src/main/res/values/dimens.xml

```diff
@@ -99,4 +99,7 @@
     <dimen name="scan_progress_margin_top">10dp</dimen>
     <dimen name="device_setup_margin_top">20dp</dimen>
 
+    <dimen name="search_skill_padding">7dp</dimen>
```

Comment:
Is it possible to use 8dp instead of 7dp?

Reply:
I tried 8dp but it looked very wide in small phones, so I think 7dp is better

Reply:
You should be using standard dimensions under material guidelines.